apply/apply.go

```diff
@@ -68,7 +69,14 @@ func execPipeline(ctx context.Context, in *graph.Graph, pipelineF MkPipelineF, r
 		notify.Transform(func(id string, out *graph.Graph) error {
 			renderingPlant.Graph = out
 			pipeline := pipelineF(out, id)
-			val, pipelineError := pipeline.Exec(out.Get(id))
```

Comment:
Since we are using the ID inside of the pipeline, does it make sense to actually go ahead and pass the entire node into the pipeline instead of extracting it, then pull the ID from the node rather than putting it into the pipeline?